policies/test.rego

```diff
@@ -0,0 +1,28 @@
+package hacbs.contract.test
+
+# Check if we have any test data is present
+deny[{"msg": msg}] {
+	not data.test
+	msg := "No test data provided"
+}
```

Comment:
Cool, that's a useful pattern since we otherwise might pass if there's no attestation for example.

Reply:
Yeah, I would not have thought of this if I didn't do it TDD-style. There are other things in here that a novice Rego writer, such as myself, would be tripped upon.